config/database.yml

```diff
@@ -1,25 +1,12 @@
-# SQLite. Versions 3.8.0 and up are supported.
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem "sqlite3"
-#
 default: &default
-  adapter: sqlite3
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
+  adapter: postgresql
+  encoding: unicode
+  pool: 5
+  timeout:
 
 development:
   <<: *default
-  database: db/development.sqlite3
-
-# Warning: The database defined as "test" will be erased and
-# re-generated from your development database when you run "rake".
-# Do not set this db to the same as development or production.
-test:
```

Comment:
@katemildner Great work with credentials! However - we still need a test database. It can have a default username and password in clear text, it's not a problem. Not having a test db would be a problem tho - your tests won't run without it. To check if it's working you can use `rake db:test:prepare`

Reply:
Committed and pushed, also working like a charm 👌🏻 
Although, I have some questions regarding different ways of configuration in database.yml, so I'd like to add it to the agenda of our next session, please :)

Reply:
Sure thing. Very happy it works!